src/platform-implementation-js/dom-driver/gmail/views/gmail-nav-item-view.ts

```diff
@@ -912,6 +912,10 @@ export default class GmailNavItemView {
     if (this._expandoElement) {
       this._expandoElement.title = `Expand ${name}`;
     }
+
+    this._element.firstElementChild?.setAttribute('data-tooltip', name);
```

Comment:
Could we add these attributes as [NavItemDescriptor](https://github.com/InboxSDK/InboxSDK/blob/c08fab3b9d85c1459633220a1fddf6bbc723e126/src/platform-implementation-js/dom-driver/gmail/views/gmail-nav-item-view.ts#L51) options so they're opt in?

With `data-tooltip-align`, for example, users may be in a rtl layout where they'd want alignment on the left.

Reply:
Totally agree, and with a value set to `null` it will allow users to not use this feature